src/notifications/contracts/model-schema.json

```diff
@@ -0,0 +1,17 @@
+{
+  "to": "String",
+  "type": "String",
+  "title": "String",
+  "message": "String",
+  "unread": {
+    "type": "Boolean",
+    "default": true
+  },
+  "sent": {
+    "type": "Boolean",
+    "default": false
+  },
```

Comment:
@jac1013 great, I added the current schema in [#14](https://github.com/bixlabs/bix-notifications/issues/14).
Are we ready to merge this PR?